application/common/models/Method.php

```diff
@@ -364,11 +368,13 @@ public static function deleteExpiredUnverifiedMethods()
                                 ->all();
 
         foreach ($methods as $method) {
-            if ( ! $method->delete()) {
+            try {
+                $method->delete();
```

Comment:
Does `$method->delete()` ever return a value, or does it exclusively depend on throwing exceptions?